Appsilon/shiny.blueprint
# Add `runExample()` function

### Changes
Closes #9:
1. Move all component examples a directory level up.
2. Implement `runExample()`.

### How to test
Install the package (`reinstall.sh`) and run `appsilon.blueprint::runExample()`. Check if `showcase` and component examples work.
